src/client/eppo-client-with-bandits.spec.ts

```diff
@@ -220,6 +223,20 @@ describe('EppoClient Bandits E2E test', () => {
       expect(mockLogAssignment).not.toHaveBeenCalled();
       expect(mockLogBanditAction).not.toHaveBeenCalled();
 
+      const repeatAssignment = client.getBanditAction(
```

Comment:
@rasendubi adjusted test that flushes events to also ensure they are deduped as well (verified later with the `.toHaveBeenCalledTimes(1)` check)